SD/FPP_Install.sh

```diff
@@ -1270,6 +1270,15 @@ case "${OSVER}" in
 esac
 
 
+#######################################
+echo "FPP - Configuring NTP Daemon"
+
+# Clear all existing servers and pools and set default pool to be falconplayer NTP Pool
+sed -i '/^server.*/d' /etc/ntp.conf 
+sed -i '/^pool.*/d' /etc/ntp.conf 
+sed -i '\$s/\$/\\npool falconplayer.pool.ntp.org iburst minpoll 8 maxpoll 12 prefer/' /etc/ntp.conf
```

Comment:
Since this isn't in a string, should the $ and \ be escaped?  I understand in the settings.php, but it doesn't seem correct here.

Reply:
Yep you're absolutely right, I found & fixed this on my test box but forgot to add the commit!@ Will get it sorted :)